packages/main/src/Card.ts

```diff
@@ -85,6 +87,24 @@ class Card extends UI5Element {
 	@slot({ type: HTMLElement, invalidateOnChildChange: true })
 	header!: Array<CardHeader>;
 
+	/**
+	 * Defines if a busy indicator would be displayed over the card.
+	 * @default false
+	 * @public
+	 * @since 2.1.0-rc.1
```

Comment:
Double check which version will this be released with: 2.1.0-rc.2 or 2.1.0-rc.3

Reply:
Done